share/picongpu/dockerfiles/ubuntu-1604/Singularity

```diff
@@ -1,10 +1,10 @@
 Bootstrap: docker
-From: ax3l/picongpu:0.5.0
+From: ax3l/picongpu:0.5.0-rc1
 
 
 %labels
 Maintainer Axel Huebl <a.huebl@hzdr.de>
```

Comment:
This seams to work here, too:
```suggestion
Maintainer "Rene Widera <r.widera@hzdr.de>, Axel Huebl <a.huebl@hzdr.de>"
```

Reply:
I hoped that it is not coming to the point that I need to maintain the docker too^^
I will work on all the docker stuff end of next week.